sql/src/main/java/org/apache/druid/sql/calcite/planner/DruidPlanner.java

```diff
@@ -790,5 +862,11 @@ public SqlNode getQueryNode()
     {
       return query;
     }
+
+    @Nullable
+    public String getIngestionGranularity()
+    {
+      return ingestionGranularity;
```

Comment:
Would recommend "semanticising" this into something structured, such as a `PERIOD`. That tells the consumer that the value has been validated and that the consumer need not do their own validation.